From: Takashi Iwai <tiwai@suse.de>
Subject: Add kABI placeholders for HD-audio
Patch-mainline: Never, kABI padding
References: bsn#129 bsc#1179531
Modified-by-SEL: No

Add a few kABI padding fields to HD-audio objects that are embedded
in other structs.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 include/sound/hdaudio.h |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/include/sound/hdaudio.h
+++ b/include/sound/hdaudio.h
@@ -92,6 +92,8 @@ struct hdac_device {
 	bool lazy_cache:1;	/* don't wake up for writes */
 	bool caps_overwriting:1; /* caps overwrite being in process */
 	bool cache_coef:1;	/* cache COEF read/write too */
+
+	void *suse_kabi_padding;	/* XXX SLE-specific kABI placeholder */
 };
 
 /* device/driver type used for matching */
@@ -370,6 +372,8 @@ struct hdac_bus {
 
 	/* factor used to derive STRIPE control value */
 	unsigned int sdo_limit;
+
+	void *suse_kabi_padding;	/* XXX SLE-specific kABI placeholder */
 };
 
 int snd_hdac_bus_init(struct hdac_bus *bus, struct device *dev,
@@ -544,6 +548,8 @@ struct hdac_stream {
 	/* DSP access mutex */
 	struct mutex dsp_mutex;
 #endif
+
+	void *suse_kabi_padding;	/* XXX SLE-specific kABI placeholder */
 };
 
 void snd_hdac_stream_init(struct hdac_bus *bus, struct hdac_stream *azx_dev,
